src/main/java/com/github/manedev79/timesheet/workingday/TimesheetController.java

```diff
@@ -19,10 +21,25 @@ public TimesheetController(WorkingDayService workingDayService) {
         this.workingDayService = workingDayService;
     }
 
+    @GetMapping(params = "date")
+    public List<WorkingDayDto> getTimesheetForMonthByDate(@RequestParam("date") final LocalDate date) {
```

Comment:
@JanLoebel do you prefer that signature? Otherwise we can go with the YearMonth as below.